.github/workflows/pr-checks.yaml

```diff
@@ -184,6 +205,26 @@ jobs:
           kubectl apply -f out/release.yaml
           make TEST_ACCEPTANCE_START_SBO=remote test-acceptance
 
+      - name: Collect Kube resources
+        continue-on-error: true
+        run: |
+          kubectl api-resources --verbs=list --namespaced -o name > resources.list
+
+          ns=service-binding-operator
+          for res in $(cat resources.list | grep -v secrets); do
+            mkdir -p ${TEST_RESULTS}/${ns}
+            kubectl get ${res} --ignore-not-found -n ${ns} -o yaml > ${TEST_RESULTS}/${ns}/${res}.yaml;
+          done
+
+          if [ -f out/test-namespace ]; then
+            ns=$(cat out/test-namespace)
+            for res in $(cat resources.list); do
+              mkdir -p ${TEST_RESULTS}/${ns}
+              kubectl get ${res} --ignore-not-found -n ${ns} -o yaml > ${TEST_RESULTS}/${ns}/${res}.yaml;
+            done
+          fi
+        if: always()
+
```

Comment:
how about that we make a custom action, so to avoid having this code twice in the workflows?

Reply:
+1

Reply:
+1